Prace_domowe/Praca_domowa2/Grupa2/Komorowski_Michal/notebook.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Jest to prawda. Oczywiście można próbować (i na niektórych maszynach może by to nawet przeszło), ale one-hot encoding nie jest optymalnym rozwiązaniem w tym przypadku. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>